x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form_objectives_timeslices.tsx

```diff
@@ -65,7 +67,7 @@ export function SloEditFormObjectivesTimeslices({ control }: Props) {
               min={1}
               max={120}
               step={1}
-              onChange={(event) => field.onChange(String(event.target.value))}
+              onChange={(event) => field.onChange(String(Number(event.target.value)))}
```

Comment:
Why do you need to use `Number` here?

Reply:
Interesting thing here. Since we are using the API response type, `timeslices duration`  is actually a string (because it is formatted as "1m" or "1h"). When we read the response, we trim the unit part and keep only the duration, but it is still a string from the type standpoint.




Therefore, here I'm parsing the string value to a Number (so the input works well), and finally to a String because that's the expected type. If I don't `Number()` it, the input does not behave like this (going from value to 0 to new value) 👇🏻 

https://user-images.githubusercontent.com/1376800/212107613-584c20ad-9ecc-4fc0-b6a8-7679a91fcc50.mov